### ISSUE TITLE
Allow sites to configure people pages by place or bounding box

### ISSUE THREAD

**Issue by loarie** [inaturalist/inaturalist#1992]
Right now, sites can configure obs/search to default to site, place, or bounding box. Lots of sites have chosen/prefer place
<img width="1186" alt="screen shot 2018-10-11 at 1 58 52 pm" src="https://user-
images.githubusercontent.com/598026/46833894-b4e00400-cd5e-11e8-8bb4-318b15af1cd4.png">
But the people page can only show site or all.

If the site is configured to have place on obs/search, then the site can't be configured to have the people page match

proposal: allow sites to configure the people page to use eithr site, place, or bounding box rather than site or all

Thoughts @carrieseltzer ?

**Comment by carrieseltzer** [comment]
Yes, I think it makes a lot of sense to allow the people page to be configured by site affiliation, place, or bounding box.
